src/Uno.UWP/Graphics/Display/DisplayInformation.Android.cs

```diff
@@ -154,31 +154,51 @@ public float RawDpiY
 		/// <see cref="https://stackoverflow.com/questions/4553650/how-to-check-device-natural-default-orientation-on-android-i-e-get-landscape">this SO question</see>
 		/// </remarks>
 		private DisplayOrientations GetNativeOrientation()
-		{
-			using (var windowManager = CreateWindowManager())
-			{
-				var orientation = ContextHelper.Current.Resources.Configuration.Orientation;
-				if (orientation == Android.Content.Res.Orientation.Undefined)
-				{
-					return DisplayOrientations.None;
-				}
-
-				var rotation = windowManager.DefaultDisplay.Rotation;
-				bool isLandscape;
-				switch (rotation)
-				{
-					case SurfaceOrientation.Rotation0:
-					case SurfaceOrientation.Rotation180:
-						isLandscape = orientation == Android.Content.Res.Orientation.Landscape;
-						break;
-					default:
-						isLandscape = orientation == Android.Content.Res.Orientation.Portrait;
-						break;
-				}
-				return isLandscape ? DisplayOrientations.Landscape : DisplayOrientations.Portrait;
-			}
-		}
-
+		{
+			var orientation = ContextHelper.Current.Resources.Configuration.Orientation;
+			if (orientation == Android.Content.Res.Orientation.Undefined)
+			{
+				return DisplayOrientations.None;
+			}
+
+			SurfaceOrientation rotation;
+
+			if (Android.OS.Build.VERSION.SdkInt <= Android.OS.BuildVersionCodes.Q)
+			{
+				using var windowManager = CreateWindowManager();
+#pragma warning disable 618
+				// WindowManager.DefaultDisplay adeprecated in API30
+				rotation = windowManager.DefaultDisplay.Rotation;
+#pragma warning restore 618
+			}
+			else
+			{
+				using var display = ContextHelper.Current.Display;

```

Comment:
It's likely that this may not be available on some android versions (likely android 10 ?), hence your build error.